Spec/RealtimeClientConnection.swift

```diff
@@ -1962,7 +1962,7 @@ class RealtimeClientConnection: QuickSpec {
                     }
                 }
 
-                it("should transition to Failed when the token renewal fails") {
+                it("should transition to disconnected when the token renewal fails") {
```

Comment:
Maybe we should add a test for the Failed case because the spec says:

> **RTN14b** ... connection will transition to the DISCONNECTED state, and the Connection#errorReason should be set. **(If no means to renew the token is provided, RSA4a applies)**

Reply:
@ricardopereira Is this what you mean? https://github.com/ably/ably-ios/blob/develop/Spec/RealtimeClientConnection.swift#L2008

Reply:
Yes, that's it 👍